plugins/modules/consul_policy.py

```diff
@@ -20,12 +21,13 @@
   - Håkon Lerring (@Hakon)
 extends_documentation_fragment:
   - community.general.consul
+  - community.general.consul.token
   - community.general.attributes
 attributes:
   check_mode:
-    support: none
+    support: full
   diff_mode:
-    support: none
+    support: partial
```

Comment:
If this is new, it needs a changelog fragment, and
```suggestion
    support: partial
    version_added: 8.3.0
```
(same for check mode)

Reply:
Also, `support: partial` should come with `details:` which accepts a list of strings (paragraphs) explaining why it's not `support: full`.